src/transforms/functional.jl

```diff
@@ -6,6 +6,18 @@
     Functional(function)
 
 The transform that applies a `function` elementwise.
+
+# Examples
+
+```julia
+Functional(cos)
+Functional(sin)
+```
+
+## Notes
+
+* The schema of the table is preserved by the transform.
+
```

Comment:
Please remove these notes. We already had this discussion in other PRs. This information is not useful.